nodemon.json

```diff
@@ -1,9 +1,7 @@
 {
```

Comment:
a sua configuração de lint pessoal, afetou o projeto, trouxe vários arquivos que estavam com 4 espaçamentos ou uma tab para 2 espaçamentos, acabou deixando o projeto com arquivos em que alguns estão com espaçamentos de 2 espaços e 4 espaços, vamos manter o padrão de 4 espaços para não ter confusões com isso posteriormente e facilitar a identificação quando o código for alterado.